src/hotspot/share/ci/ciMethodBlocks.cpp

```diff
@@ -33,6 +33,7 @@
 
 
 ciBlock *ciMethodBlocks::block_containing(int bci) {
+  assert(bci >=0 && bci < _code_size, "valid bytecode range");
```

Comment:
```suggestion
  assert(bci >= 0 && bci < _code_size, "valid bytecode range");
```

Reply:
I copied that from the code below.  I will fix both.